Carbon/Carbon.psd1

```diff
@@ -359,96 +359,13 @@ All functions are idempotent: when run multiple times with the same arguments, y
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
-# 2.10.2
 
-* Fixed: Carbon fails to import if IIS isn't installed.
+# 2.11.0
 
-# 2.10.1
-
-* Fixed: Carbon fails to import on PowerShell 4.
-
-# 2.10.0
+* New: 'Grant-Permission', 'Get-Permission', and 'Revoke-Permission' scripts now execute correctly on
+* non-Windows platforms.

```

Comment:
This is Markdown syntax, so no `*` on the second line, since it continues from the first line.

Reply:
The extra `*` is still here.